themes/ddbasic/template.node.php

```diff
@@ -136,6 +136,13 @@ function ddbasic_preprocess__node__ding_event(&$variables) {
     $variables['event_price'] = t('Free');
   }
 
+  $location = field_get_items('node', $variables['node'], 'field_ding_event_location');
+  if(!empty($location[0]['name_line']) || !empty($location[0]['thoroughfare'])) {
+    $variables['alt_location_is_set'] = TRUE;
+  } else {
+    $variables['alt_location_is_set'] = FALSE;
+  }
```

Comment:
This could be simplified to 

```php
$variables['alt_location_is_set'] = !empty($location[0]['name_line']) || !empty($location[0]['thoroughfare']);
```